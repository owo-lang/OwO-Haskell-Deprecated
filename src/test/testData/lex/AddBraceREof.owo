postulate
 a
 postulate
  b
  postulate
   c
   postulate
    d
    postulate
     e
     postulate
      f
