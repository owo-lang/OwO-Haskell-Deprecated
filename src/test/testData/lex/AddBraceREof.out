PostulateToken (0 1 1) (9 1 10)
BraceLToken (11 2 2) (11 2 2)
IdentifierToken "a" (11 2 2) (12 2 3)
SemicolonToken (14 3 2) (14 3 2)
PostulateToken (14 3 2) (23 3 11)
BraceLToken (26 4 3) (26 4 3)
IdentifierToken "b" (26 4 3) (27 4 4)
SemicolonToken (30 5 3) (30 5 3)
PostulateToken (30 5 3) (39 5 12)
BraceLToken (43 6 4) (43 6 4)
IdentifierToken "c" (43 6 4) (44 6 5)
SemicolonToken (48 7 4) (48 7 4)
PostulateToken (48 7 4) (57 7 13)
BraceLToken (62 8 5) (62 8 5)
IdentifierToken "d" (62 8 5) (63 8 6)
SemicolonToken (68 9 5) (68 9 5)
PostulateToken (68 9 5) (77 9 14)
BraceLToken (83 10 6) (83 10 6)
IdentifierToken "e" (83 10 6) (84 10 7)
SemicolonToken (90 11 6) (90 11 6)
PostulateToken (90 11 6) (99 11 15)
BraceLToken (106 12 7) (106 12 7)
IdentifierToken "f" (106 12 7) (107 12 8)
BraceRToken (108 13 1) (108 13 1)
BraceRToken (108 13 1) (108 13 1)
BraceRToken (108 13 1) (108 13 1)
BraceRToken (108 13 1) (108 13 1)
BraceRToken (108 13 1) (108 13 1)
BraceRToken (108 13 1) (108 13 1)
